### ISSUE TITLE
Convert Deployment Considerations to MD

### ISSUE THREAD

**Issue by JustinCappos** [uptane/uptane-standard#68]
The google doc containing deployment considerations needs to be merged into markdown and added to github so that we can easily collaboratively edit / comment on changes.  Simply making edits to the doc (as we have been), isn't the right model.  We should be doing more rigorous reviews as we did for the standards doc.

I read through the document and tried to resolve all of the open comments in a sane way in preparation for this effort.  

If possible could one / both of @patrickvacek and @jhdalek55 help to make this happen?

**Comment by jhdalek55** [comment]
I agree that duplication of effort at this point is a waste of time. If 
most of the existing document will be eventually serve as the raw text 
for the deployment pages, why don't we just use that text to fill in the 
blanks. In other words, why don't I just cut and paste the existing text 
onto the web pages?


If that's is needed right now, I can take care of that this week.


Lois

On Sat, Mar 02, 2019 at 10:51 AM, Justin Cappos wrote:

The google doc containing deployment considerations needs to be merged 
into markdown and added to github so that we can easily collaboratively 
edit / comment on changes.  Simply making edits to the doc (as we have 
been), isn't the right model.  We should be doing more rigorous reviews 
as we did for the standards doc.

I read through the document and tried to resolve all of the open 
comments in a sane way in preparation for this effort.

If possible could one / both of @patrickvacek 
<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_patrickvacek&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=hgBKIqNYIOwzXeBjPUaKRw&m=RhlXarMt02vmjmHSZzgZnwloOuTmd39D3C-thBjS1X8&s=qjwDGqIoieYI4DW8TjiilRdW7lLIsnjJEY1tLmrGuPM&e=> 
and @jhdalek55 
<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_jhdalek55&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=hgBKIqNYIOwzXeBjPUaKRw&m=RhlXarMt02vmjmHSZzgZnwloOuTmd39D3C-thBjS1X8&s=LvBOqqyZ-QSjxeLvIZVY6FIpEQewp1Ub1qCYsQEoue4&e=> 
help to make this happen?


—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub 
<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_uptane_uptane-2Dstandard_issues_68&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=hgBKIqNYIOwzXeBjPUaKRw&m=RhlXarMt02vmjmHSZzgZnwloOuTmd39D3C-thBjS1X8&s=pthWaJFRj9eaaxlekfFX43B6Hla3iw7xeAGQBe-bcbo&e=> 
, or mute the thread 
<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AN5qXy1GFqsiNlvHhGCu8NnFlI0YAXOgks5vSp3tgaJpZM4bac-5FB&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=hgBKIqNYIOwzXeBjPUaKRw&m=RhlXarMt02vmjmHSZzgZnwloOuTmd39D3C-thBjS1X8&s=zGfjT_Plcq6vfOewZbXXhC-JustLqWTstcf-tXqpXqg&e=> 
.

**Comment by JustinCappos** [comment]
Yes, this would be helpful.

Also, forgive my bad memory, but did we resolve to make these into webpages or into a separate standards doc?  I recall the discussion but do not recall the outcome.

**Comment by jhdalek55** [comment]
Webpages, for which Jon Oster has already set up individual markdown 
files and established subheads. I can just cut and paste in the text and 
then the commitee can review the individual entries.


I can get this done the early part of next week.


Lois

On Sat, Mar 02, 2019 at 11:29 AM, Justin Cappos wrote:

Yes, this would be helpful.

Also, forgive my bad memory, but did we resolve to make these into 
webpages or into a separate standards doc?  I recall the discussion but 
do not recall the outcome.


—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub 
<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_uptane_uptane-2Dstandard_issues_68-23issuecomment-2D468935777&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=hgBKIqNYIOwzXeBjPUaKRw&m=cH2ZzW87ly-JCkegt3puhjdRyEyVoyWwQcDjLjOqJoY&s=qlQnfUwrGIHcMksBRpi4fFUndeo4XFBhsrOzpq2lqCs&e=> 
, or mute the thread 
<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AN5qX4RLFBnE4CDgmYwPvt6AaE86waTVks5vSqbpgaJpZM4bac-5FB&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=hgBKIqNYIOwzXeBjPUaKRw&m=cH2ZzW87ly-JCkegt3puhjdRyEyVoyWwQcDjLjOqJoY&s=qUgjtf3oZXUnXNmUKXGrzA7bKJsMlPJ7XUl1hKqP_XY&e=> 
.

**Comment by JustinCappos** [comment]
Okay, sounds good.  FYI: I added an E.8 this morning, and I imagine other heading names will be added, removed, 
 or revised over time.

**Comment by jhdalek55** [comment]
No problem. I'll add files if needed.


Lois

On Sat, Mar 02, 2019 at 11:42 AM, Justin Cappos wrote:

Okay, sounds good.  FYI: I added an E.8 this morning, and I imagine 
other heading names will be added, removed,

or revised over time.


—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub 
<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_uptane_uptane-2Dstandard_issues_68-23issuecomment-2D468937023&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=hgBKIqNYIOwzXeBjPUaKRw&m=k77udPygIa8oDzs8vdUzjs3fcAv3HbfwFqjG-LeBXjc&s=EkfbYEarSrafwPJ3_xMAL6jvMr2fgpfSkjQBEkBS0vU&e=> 
, or mute the thread 
<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AN5qX8tnBl31PXcwK2oB0H1EENG923wKks5vSqn8gaJpZM4bac-5FB&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=hgBKIqNYIOwzXeBjPUaKRw&m=k77udPygIa8oDzs8vdUzjs3fcAv3HbfwFqjG-LeBXjc&s=6pRP3LHs2dk3WYHE69U3jIRW6WXpZH_XifyJRzuDefM&e=> 
.

**Comment by tkfu** [comment]
On the 03/13 standards call, we discussed closing this and moving it to https://github.com/uptane/uptane.github.io/, but didn't reach a conclusion. Discussion about the most appropriate place/format for deployment considerations will continue on the mailing list.

**Comment by jhdalek55** [comment]
I'm planning to begin moving the Deployment materials to the markdown files today. Before I begin, I did want to make a few comments about formatting issues in  the current Google Doc version of Deployment Considerations.

First, there are a large number of figures in the document. My experience in embedding figures in Markdown is somewhat limited. Also, I'm assuming these figures may require updating anyway as different employment scenarios are considered. So, at this point, particularly while we are still discussing the best way to present the deployment material, I would rather leave revision and placement of the figures to others who are more knowledgeable about the subject matter and about the best way to position these figures.

The second issue is that the document involves a great deal of crosslinking, either to other parts of the document or to other documents. I am not going to attempt such crosslinks until the question of document formatting is settled.

In short, I'm planning to just move raw text over so we can begin editing and rewriting as required.

I hope this is agreeable at this point.

Lois

**Comment by trishankatdatadog** [comment]
Of course, Lois. I can help with the figures later. Thanks so much for doing this! @jhdalek55

**Comment by iramcdonald** [comment]
Hi Lois,

Crosslinking:  In the recent SAE J3101 Hardware Protected Security
Environment Requirements
spec (120 pages), we finally removed *all* crosslinking by active links
and/or explicit section numbers
and replaced it with *names* of referenced sections only - we did this on
the advice of the SAE staff
technical writers (who will soon take over the J3101 draft for publication
in summer 2019).

Cheers,
- Ira

Ira McDonald (Musician / Software Architect)
Co-Chair - TCG Trusted Mobility Solutions WG
Co-Chair - TCG Metadata Access Protocol SG
Chair - Linux Foundation Open Printing WG
Secretary - IEEE-ISTO Printer Working Group
Co-Chair - IEEE-ISTO PWG Internet Printing Protocol WG
IETF Designated Expert - IPP & Printer MIB
Blue Roof Music / High North Inc
http://sites.google.com/site/blueroofmusic
http://sites.google.com/site/highnorthinc
mailto: blueroofmusic@gmail.com
PO Box 221  Grand Marais, MI 49839  906-494-2434

**Comment by patrickvacek** [comment]
The current crosslinking scheme for the Uptane standard works great IMO. You can link by section name or section number, and it is easy to maintain. For the deployment considerations, it seems essential to have crosslinking, especially if we deploy them as a website.

However, Lois, I understand that at the moment that shouldn't be the highest priority. I see no problem with you copying text as you see fit, and then we can do a pass or two later to re-establish the links as necessary.

**Comment by jhdalek55** [comment]
On the 3/27 conference call @jhdalek55 reported that she has begun making this transfer and hopes toc complete it in the next week or so.. She met with @tkfu  and @patrickvacek after the regular to discuss a few issues related to this transfer.

**Comment by tkfu** [comment]
Moved to https://github.com/uptane/uptane.github.io/issues/11.
